src/google-maps/map-advanced-marker/map-advanced-marker.ts

```diff
@@ -0,0 +1,255 @@
+   */
+  @Input()
+  set title(title: string) {
+    this._title = title;
+  }
+  private _title: string;
+
+  /**
+   * Sets the AdvancedMarkerElement's position. An AdvancedMarkerElement may be constructed without a position, but will not be displayed until its position is provided - for example, by a user's actions or choices. An AdvancedMarkerElement's position can be provided by setting AdvancedMarkerElement.position if not provided at the construction.
+   * Note: AdvancedMarkerElement with altitude is only supported on vector maps.
+   * https://developers.google.com/maps/documentation/javascript/reference/advanced-markers#AdvancedMarkerElementOptions.position
+   */
+  @Input()
+  set position(
+    position:
+      | google.maps.LatLngLiteral
+      | google.maps.LatLng
+      | google.maps.LatLngAltitude
+      | google.maps.LatLngAltitudeLiteral,
+  ) {
+    this._position = position;
+  }
+  private _position: google.maps.LatLngLiteral | google.maps.LatLng;
+
+  /**
+   * The DOM Element backing the visual of an AdvancedMarkerElement.
+   * Note: AdvancedMarkerElement does not clone the passed-in DOM element. Once the DOM element is passed to an AdvancedMarkerElement, passing the same DOM element to another AdvancedMarkerElement will move the DOM element and cause the previous AdvancedMarkerElement to look empty.
+   * See: https://developers.google.com/maps/documentation/javascript/reference/advanced-markers#AdvancedMarkerElementOptions.content
+   */
+  @Input()
+  set content(content: Node | google.maps.marker.PinElement) {
+    this._content = content;
+  }
+  private _content: Node;
+
+  /**
+   * If true, the AdvancedMarkerElement can be dragged.
+   * Note: AdvancedMarkerElement with altitude is not draggable.
+   * https://developers.google.com/maps/documentation/javascript/reference/advanced-markers#AdvancedMarkerElementOptions.gmpDraggable
+   */
+  @Input()
+  set gmpDraggable(draggable: boolean) {
+    this._draggable = draggable;
+  }
+  private _draggable: boolean;
+
+  /**
+   * Options for constructing an AdvancedMarkerElement.
+   * https://developers.google.com/maps/documentation/javascript/reference/advanced-markers#AdvancedMarkerElementOptions
+   */
+  @Input()
+  set options(options: google.maps.marker.AdvancedMarkerElementOptions) {
+    this._options = options;
+  }
+  private _options: google.maps.marker.AdvancedMarkerElementOptions;
+
+  /**
+   * All AdvancedMarkerElements are displayed on the map in order of their zIndex, with higher values displaying in front of AdvancedMarkerElements with lower values. By default, AdvancedMarkerElements are displayed according to their vertical position on screen, with lower AdvancedMarkerElements appearing in front of AdvancedMarkerElements farther up the screen. Note that zIndex is also used to help determine relative priority between CollisionBehavior.OPTIONAL_AND_HIDES_LOWER_PRIORITY Advanced Markers. A higher zIndex value indicates higher priority.
```

Comment:
I think that we can just say that it's the Z index of the map and link to the docs, no need to paste the entire description.

Reply:
I've changed it to shorter description.